ansible/roles/service_catalog_setup/tasks/main.yml

```diff
@@ -26,6 +26,10 @@
     shell: "{{ oc_cmd }} get project --no-headers=true | awk '{ print $1 }' | grep -E '^service-catalog( |$)' | cat"
     register: project
 
+  - name: check if the service-catalog ssl secret exists
+    shell: "{{ oc_cmd }} get secret --no-headers=true | awk '{ print $1 }' | grep -E '^apiserver-ssl( |$)' | cat"
```

Comment:
needs a `-n service-catalog` to  check the correct namespace otherwise reruns fails.